{"level":"error","message":"Signup error Cannot read properties of undefined (reading 'body')","service":"acquisitions-api","stack":"TypeError: Cannot read properties of undefined (reading 'body')\n    at signup (file:///C:/Users/USER/WebstormProjects/acquisitions/src/controllers/auth.controller.js:11:33)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:295:15\n    at processParams (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:582:12)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:291:5)\n    at Function.handle (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:186:3)"}
{"level":"error","message":"Error creating the user: ReferenceError: db is not defined","service":"acquisitions-api"}
{"level":"error","message":"Signup error db is not defined","service":"acquisitions-api","stack":"ReferenceError: db is not defined\n    at createUser (file:///C:/Users/USER/WebstormProjects/acquisitions/src/services/auth.service.js:16:30)\n    at signup (file:///C:/Users/USER/WebstormProjects/acquisitions/src/controllers/auth.controller.js:21:28)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:295:15\n    at processParams (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:582:12)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:291:5)"}
{"level":"error","message":"Error creating the user: ReferenceError: eq is not defined","service":"acquisitions-api"}
{"level":"error","message":"Signup error eq is not defined","service":"acquisitions-api","stack":"ReferenceError: eq is not defined\n    at createUser (file:///C:/Users/USER/WebstormProjects/acquisitions/src/services/auth.service.js:21:13)\n    at signup (file:///C:/Users/USER/WebstormProjects/acquisitions/src/controllers/auth.controller.js:21:28)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\lib\\layer.js:152:17)\n    at C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:295:15\n    at processParams (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:582:12)\n    at next (C:\\Users\\USER\\WebstormProjects\\acquisitions\\node_modules\\router\\index.js:291:5)"}
{"level":"error","message":"Authentication error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","service":"acquisitions-api"}
{"cause":{"name":"NeonDbError","sourceError":{"cause":{"address":"172.18.0.2","code":"ECONNREFUSED","errno":-111,"port":443,"syscall":"connect"}}},"level":"error","message":"Sign-in error Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","params":["john@test.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///app/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///app/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async authenticateUser (file:///app/src/services/auth.service.js:28:30)\n    at async signin (file:///app/src/controllers/auth.controller.js:64:22)"}
{"level":"error","message":"Error creating the user: Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","service":"acquisitions-api"}
{"cause":{"name":"NeonDbError","sourceError":{"cause":{"address":"172.18.0.2","code":"ECONNREFUSED","errno":-111,"port":443,"syscall":"connect"}}},"level":"error","message":"Signup error Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","params":["john@test.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///app/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///app/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///app/src/services/auth.service.js:61:30)\n    at async signup (file:///app/src/controllers/auth.controller.js:21:22)"}
{"level":"error","message":"Error creating the user: Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","service":"acquisitions-api"}
{"cause":{"name":"NeonDbError","sourceError":{"cause":{"address":"172.18.0.2","code":"ECONNREFUSED","errno":-111,"port":443,"syscall":"connect"}}},"level":"error","message":"Signup error Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1","params":["john@test.com",1],"query":"select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"acquisitions-api","stack":"Error: Failed query: select \"id\", \"name\", \"email\", \"password\", \"role\", \"created_at\", \"updated_at\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: john@test.com,1\n    at NeonHttpPreparedQuery.queryWithCache (file:///app/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///app/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///app/src/services/auth.service.js:61:30)\n    at async signup (file:///app/src/controllers/auth.controller.js:21:22)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"acquisitions-api"}
{"level":"error","message":"Signup error User with this email already exists","service":"acquisitions-api","stack":"Error: User with this email already exists\n    at createUser (file:///app/src/services/auth.service.js:68:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///app/src/controllers/auth.controller.js:21:22)"}
{"level":"error","message":"Error getting user by ID: User not found","service":"acquisitions-api","stack":"Error: User not found\n    at getUserById (file:///app/src/services/users.services.js:39:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///app/src/controllers/users.controller.js:38:22)"}
{"level":"error","message":"Error in getUserById controller: User not found","service":"acquisitions-api","stack":"Error: User not found\n    at getUserById (file:///app/src/services/users.services.js:39:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///app/src/controllers/users.controller.js:38:22)"}
{"level":"error","message":"Error getting user by ID: User not found","service":"acquisitions-api","stack":"Error: User not found\n    at getUserById (file:///app/src/services/users.services.js:39:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///app/src/controllers/users.controller.js:38:22)"}
{"level":"error","message":"Error in getUserById controller: User not found","service":"acquisitions-api","stack":"Error: User not found\n    at getUserById (file:///app/src/services/users.services.js:39:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async getUserById (file:///app/src/controllers/users.controller.js:38:22)"}
